xapian-letor/bin/questletor.cc

```diff
@@ -166,7 +166,7 @@ try {
     parser.set_stemming_strategy(Xapian::QueryParser::STEM_SOME);
     parser.set_stopper(&mystopper);
```

Comment:
Don't we want to just delete this file?  It's been replaced by `xapian-rank.cc` hasn't it?